Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -343,7 +381,19 @@ bool HasStereoCamera()
 
         protected virtual void PerformUpdate()
         {
-            SetLocalTransform(m_CurrentPosition, m_CurrentRotation);
+            InputTrackingState currentTrackingStateFlags = InputTrackingState.None;
+            bool hasTrackingStateFlags = false;
```

Comment:
Minor, but why not direct evaluate here to reduce this code? E.g.
`bool hasTrackingStateFlags = m_TrackingStateFlags?.action?.bindings.Count > 0;
var currentTrackingStateFlags = hasTrackingStateFlags ? (InputTrackingState)m_TrackingStateFlags.action.ReadValue<int>() : InputTrackingState.None;`
or even evaluate in call to SetLocalTransform and skip the locals?

Reply:
Maybe easier to debug this way....